anvil/src/config.rs

```diff
@@ -374,6 +389,20 @@ impl NodeConfig {
         self
     }
 
+    /// Returns the genesis timestamp to use
+    pub fn get_genesis_timestamp(&self) -> u64 {
+        self.genesis_timestamp.unwrap_or_default()
```

Comment:
this will be `0` for `None`,

while this is not possible rn, since we initialize this as `Some(duration_since_unix_epoch().as_secs())`,
I'd rather initialize with None and compute move `duration_since_unix_epoch().as_secs()` into `unwrap_or_else(||)`